tests/fixtures/cat-subdiff-x-cat-subdiff.json

```diff
@@ -0,0 +1,292 @@
+{
+    "element": "shoji:view",
+    "self": "https://app.crunch.io/api/datasets/95ba1b/cube/?query=%7B%22dimensions%22%3A%5B%7B%22variable%22%3A%22https%3A%2F%2Fapp.crunch.io%2Fapi%2Fdatasets%2F95ba1b%2Fvariables%2F000000%2F%22%7D%2C%7B%22variable%22%3A%22https%3A%2F%2Fapp.crunch.io%2Fapi%2Fdatasets%2F95ba1b%2Fvariables%2F000001%2F%22%7D%5D%2C%22measures%22%3A%7B%22count%22%3A%7B%22function%22%3A%22cube_count%22%2C%22args%22%3A%5B%5D%7D%7D%2C%22weight%22%3Anull%7D&filter=%7B%7D",
+    "value": {
```

Comment:
I think cat-x-cat can be used instead of this one, can't it? Of course the expectation would change, but in general there is nothing substantially different about this variant as far as I can tell.

Reply:
Yeah, I kind of knew you'd want to use it. It was nice to have a file with real interpretations for working with it, but I can change before merging.